pkg/workloads/workload.go

```diff
@@ -244,7 +244,7 @@ const (
 		` } }`
 
 	duplicateWorkloadMutation = `
-		mutation($accountId: Int!, $sourceGuid: EntityGuid!, $workload: WorkloadDuplicateInput!) {
+		mutation($accountId: Int!, $sourceGuid: EntityGuid!, $workload: WorkloadDuplicateInput) {
```

Comment:
Does the `!` here just mean the field is required?

Reply:
Means "non-nullable", so basically yeah